react/features/stream-effects/virtual-background/JitsiStreamBackgroundEffect.js

```diff
@@ -113,7 +113,14 @@ export default class JitsiStreamBackgroundEffect {
 
         this._outputCanvasCtx.globalCompositeOperation = 'destination-over';
         if (this._options.virtualBackground.isVirtualBackground) {
-            this._outputCanvasCtx.drawImage(this._virtualImage, 0, 0);
+            this._virtualImage.style.objectFit = 'cover';
```

Comment:
As discussed, this won't work and we'll work on a proper "object-fit cover" rendering of the background image in the canvas.